### PR TITLE
feat(FOROME-882): add doc nav links to main page

### PR DESCRIPTION
**Pull request by Lapk1n**
https://quantori.atlassian.net/browse/FOROME-882

In addition:
1. Fixed bug with a jumping height of datasets list

### CODE REVIEW

**Review by AlMaQntr — APPROVED** [review]

**Review by QSdmitrioul — APPROVED** [review]

File: src/pages/main/components/doc-links.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { observer } from 'mobx-react-lite'
+
+export const DocLinks = observer(() => (
```
Comment (AndreyUstyumenko) [inline src/pages/main/components/doc-links.tsx]: nothing to observe

File: src/pages/main/ui/datasets-list.tsx [hunk 2]
```diff
@@ -1,22 +1,14 @@
...
-
-  const offsetTop = newRef?.current?.offsetTop || 0
-
-  return (
+export const DatasetsList = observer(
+  (): ReactElement => (
     <div
-      ref={newRef}
-      style={{ height: height - offsetTop }}
+      style={{ height: 'calc(100vh - 260px)' }}
```
Comment (AndreyUstyumenko) [inline src/pages/main/ui/datasets-list.tsx]: give docLinks a height and summarize it with offsetTop

File: src/pages/main/components/doc-links.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
+export const DocLinks = () => (
+  <div style={{ height: 69 }}>
```
Comment (AndreyUstyumenko) [inline src/pages/main/components/doc-links.tsx]: export const docLinksHeight = 69 >> constants.ts

File: src/pages/main/ui/datasets-list.tsx [hunk 4]
```diff
@@ -9,14 +9,22 @@ import { DatasetsListItem } from './datasets-list-item'
 export const DatasetsList = observer((): ReactElement => {
   const { height } = useWindowDimensions()
 
-  const newRef = useRef<any>(null)
+  const [offsetTop, setOffsetTop] = useState<number>(0)
 
-  const offsetTop = newRef?.current?.offsetTop || 0
+  const newRef = useRef<HTMLDivElement>(null)
+
+  const docLinksHeight = 69
```
Comment (AndreyUstyumenko) [inline src/pages/main/ui/datasets-list.tsx]: import docLinksHeight from '..'

**Review by AndreyUstyumenko — APPROVED** [review]

**Pull request merged**